guides/v2.1/cloud/project/project-patch.md

```diff
@@ -10,12 +10,12 @@ functional_areas:
 
 You can apply patches to update {{site.data.var.ece}}. We recommend using a new active branch and Integration environment for applying and testing the patch prior to fully deploying across all environments. We strongly recommend you test patches locally so you can identify and resolve any issues.
 
-The `composer update` command performs a {{site.data.var.ece}} upgrade with patches and hotfixes. Check the `magento/magento-cloud-configuration/patches` folder for available patches.
+The `composer update` command performs a {{site.data.var.ece}} upgrade with patches and hot fixes. Check the `magento/magento-cloud-configuration/patches` folder for available patches.
```

Comment:
Have we standardized on "hot fix"? If so, please have this added to the acceptable word list.

Reply:
Word check in the team:  hotfix over hot fix. Would change those back. Adding to style guide.